ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/views/PaywallBackground.kt

```diff
@@ -0,0 +1,32 @@
+package com.revenuecat.purchases.ui.revenuecatui.views;
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.BlurredEdgeTreatment
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.blur
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.RemoteImage
+import com.revenuecat.purchases.ui.revenuecatui.backgroundUrlString
+import com.revenuecat.purchases.ui.revenuecatui.extensions.conditional
+
+@Composable
+internal fun PaywallBackground(data: PaywallData) {
+    data.backgroundUrlString?.let {
+        RemoteImage(
```

Comment:
Just a question, but do we need to specify a `ContentScale` in case the image doesn't match the screen? I'm not sure if we want to `Crop` it, or just `Fit` (default). See: https://developer.android.com/jetpack/compose/graphics/images/customize#content-scale

Reply:
Good call! This should be `Crop`. Changed.